app/lib/apply_api/access_token.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module ApplyApi
+  class AccessToken
+    # TODO: Make it real!
+    def self.fetch
```

Comment:
I expect we'll issue a single access token for all Register rather than having an auth endpoint

Reply:
Nice 👌 will remove this class.